### ISSUE TITLE
Float16 not support on Intel Macs

### ISSUE THREAD

**Issue by pusateri** [BlockchainCommons/BCSwiftDCBOR#1]
```
% sw_vers
ProductName:		macOS
ProductVersion:		13.2.1
BuildVersion:		22D68
```

```
% xcodebuild -version 
Xcode 14.2
Build version 14C18
```

```
% git log
commit b2fc439f9a2cb33589c628f1688d1b10abff04ab (HEAD -> master, tag: 0.8.0, origin/master, origin/HEAD)
Author: Wolf McNally <wolf@wolfmcnally.com>
Date:   Tue Mar 21 02:00:16 2023 -0700

    Removed requirement that map entries be non-null.
```

```
% swift build
Building for debugging...
/Users/pusateri/Projects/BCSwiftDCBOR/Sources/DCBOR/Float.swift:142:11: error: 'Float16' is unavailable in macOS
extension Float16: CBORCodable {
          ^~~~~~~
Swift.Float16:4:23: note: 'Float16' has been explicitly marked unavailable here
@frozen public struct Float16 {
                      ^
/Users/pusateri/Projects/BCSwiftDCBOR/Sources/DCBOR/Decode.swift:185:41: error: argument passed to call that takes no arguments
            let f = Float16(bitPattern: UInt16(value))
                           ~~~~~~~~~~~~~^~~~~~~~~~~~~~
/Users/pusateri/Projects/BCSwiftDCBOR/Sources/DCBOR/Decode.swift:185:41: error: argument passed to call that takes no arguments
            let f = Float16(bitPattern: UInt16(value))
                           ~~~~~~~~~~~~~^~~~~~~~~~~~~~
/Users/pusateri/Projects/BCSwiftDCBOR/Sources/DCBOR/Float.swift:142:11: error: 'Float16' is unavailable in macOS
extension Float16: CBORCodable {
          ^~~~~~~
Swift.Float16:4:23: note: 'Float16' has been explicitly marked unavailable here
@frozen public struct Float16 {
                      ^

... shortened for brevity
```

**Comment by ChristopherA** [comment]
Changed name "build fails on Ventura" to  "Float16 not support on Intel Macs" based on initial evaluation by @wolfmcnally.

**Comment by pusateri** [comment]
Ahh, that makes more sense. Found this library that I will try:

[https://github.com/philipturner/swift-intel-float16]()

**Comment by wolfmcnally** [comment]
@pusateri This should now be addressed as of version 0.8.1. Please open a new issue if it does not work for you.

**Comment by pusateri** [comment]
Yes, this compiles now on Intel Mac. Thanks!
